docs/source/conf.py

```diff
@@ -51,8 +51,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'sir'
-copyright = u'2014 Wieland Hoffmann'
+project = 'sir'
+copyright = '2025 MetaBrainz Foundation'
```

Comment:
Amending https://github.com/metabrainz/sir/pull/167/commits/80e08cb326769ec5fffc065280b99f00be543fdf#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L98 to append (rather than replace) author credits.
Date ranges from `git log docs/source/conf.py`.

```suggestion
copyright = '2014-2016, Wieland Hoffmann; 2017-2025 MetaBrainz Foundation'
```